index.js

```diff
@@ -76,9 +79,15 @@ module.exports = function (config) {
   }
 
   if (!config.disableWeb) {
+    const secure = {
```

Comment:
I like this grouping of the options together. Makes me think that `web.start()`'s arguments should be refactored soon.